building/upload-debs/conf/updates

```diff
@@ -0,0 +1,4 @@
+Name: homeworld
+Components: main
+Method: http://web.mit.edu/hyades/debian/
+VerifyRelease: 9EE78588
```

Comment:
We should never use an eight-character hash to verify anything, if we can help it. Can we use a 16-character hash here?